R/utils.R

```diff
@@ -1,53 +1,3 @@
-#' Panel group widget
```

Comment:
No longer used.

It was being used by this PR in the start and noticed that this internal function is not being used anymore